### PR TITLE
feat: update metrics

### PR DESCRIPTION
**Pull request by davidebianchi**

### CODE REVIEW

File: main_test.go [hunk 1]
```diff
@@ -1976,7 +1981,7 @@ filter_policy {
 		require.Equal(t, http.StatusOK, w.Result().StatusCode)
 
 		responseBody := getResponseBody(t, w)
-		require.Contains(t, string(responseBody), "go_gc_duration_seconds")
+		require.Contains(t, string(responseBody), "promhttp_metric_handler_errors_total")
```
Comment (fredmaggiowski) [inline main_test.go:1984]: Why this change?

File: metrics/metrics.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+const (
+	Prefix = "rond"
+
+	PolicyEvalDuration = "policy_evaluation_duration_milliseconds"
```
Comment (fredmaggiowski) [inline metrics/metrics.go:20]: Policy eval durationmetricname?

File: metrics/prometheus/prom_test.go [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rondprometheus
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/prometheus/client_golang/prometheus"
```
Comment (fredmaggiowski) [inline metrics/prometheus/prom_test.go:21]: Import order

**Review by fredmaggiowski — APPROVED** [review]
Some minor comments but in the meantime LGTM

File: main_test.go (hunk 1 above)
Comment (davidebianchi) [reply main_test.go:1984]: go collector and process collector setup are moved on main file, but in this test we setup the router. So, those metrics are not exposed. I change the test to check that the PolicyEvalDuration metric is exposed correctly
